### DISCUSSION TITLE
Quais libs oferecem componentes sem estilos facilmente estilizáveis e acessíveis por padrão?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2246] [Discussão]
Me deparei mais uma vez em busca de um dropdown acessível que fosse fácil de aplicar estilos com classes de CSS clássico.

Encontrei um tipo libs que oferecem componentes sem nenhum estilo:

- Headless UI - Headless UI – Unstyled, fully accessible UI components https://headlessui.dev/
- Rasdix UI - An open-source UI component library for building high-quality, accessible design systems and web apps. https://www.radix-ui.com/

Principais requisitos que procurei:

- Poder importar os componentes separadamente, para economizar no tamanho do bundle final.
- Acessível por padrão.
- Facilmente estilizavel com CSS clássico, classes globais, herança etc.. ou técnicas mais complexas de scoping.

**Outras opções parecidas mas com problemas**
- CSSUI https://www.cssui.dev/docs/getting-started/introduction/

Radix foi o que mais me agradou, encontrar uma lib dessas não é tão simples como fazer uma busca na internet, por isso eu trago este assunto ao fórum para poder aprender com a experiência de vocês diretamente, conhecem alguma lib que oferece componentes acessíveis e que são facilmente estilizáveis com CSS clássico? Independentemente de framework.

Agradeço desde já!

**Comment by igorthurow** [comment]
Daora mano. Bom saber.
